x11-misc/picom/picom-8.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+inherit meson python-r1 xdg
```

Comment:
With the conversion script removed, no need anymore for PYTHON_COMPAT and python-r1 eclass

Reply:
Changed to python-any-r1 in an attempt to get tests working. Unfortunately some tests need python-dbus-next, which isn't packaged, so I `RESTRICT=test`.